pydis_site/apps/content/resources/guides/pydis-guides/help-channel-guide.md

```diff
@@ -42,14 +42,14 @@ There are always 3 available help channels waiting to be claimed in the **Python
 
 In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **Python Help: Occupied** category.
 
-If you're unable to type into these channels, this means you're currently **on cooldown**. In order to prevent someone from claiming all the channels for themselves, **we only allow someone to claim a new help channel every 15 minutes**. However, if you close your help channel using the `!dormant` command, this cooldown is reset early.
+If you're unable to type into these channels, this means you're currently **on cooldown**. In order to prevent someone from claiming all the channels for themselves, **we only allow someone to claim a single help channel at the same time**. You can search in the top right corner of your Discord client `from:yourusername#xxxx` to find back your channel.
 
 ![Channel available message](/static/images/content/help_channels/available_message.png)
```

Comment:
The message has since been updated, so this image should be updated to reflect the new message.

Reply:
Addressed in https://github.com/python-discord/site/pull/624/commits/32db9c54f46edbca8036f6cf2e7dc94ac78ad056